models/dodo/arbitrum/dodo_aggregator_arbitrum_trades.sql

```diff
@@ -20,78 +21,79 @@ WITH dexs AS
         -- dodo proxy
         SELECT
             evt_block_time AS block_time,
-            'DODO' AS project,
+            'DODO X' AS project,
             '0' AS version,
             sender AS taker,
-            '' AS maker,
+            CAST(NULL AS VARBINARY) AS maker,
             fromAmount AS token_bought_amount_raw,
             returnAmount AS token_sold_amount_raw,
             cast(NULL as double) AS amount_usd,
             fromToken AS token_bought_address,
             toToken AS token_sold_address,
             contract_address AS project_contract_address,
             evt_tx_hash AS tx_hash,
-            CAST(ARRAY() as array<bigint>) AS trace_address,
+            CAST(NULL AS ARRAY<bigint>) AS trace_address,
```

Comment:
i think rather than force null and have incremental logic break, we should standardize to match other models in the `dex_aggregator.trades` lineage to default to `ARRAY[-1] as trace_address`, then re-add to the unique key, so that it's consistent across all models in lineage

this is based on other aggregators i've seen migrated recently, where the -1 was used

Reply:
Fixed it